test/simulator/misc/statusmoves.js

```diff
@@ -32,31 +32,51 @@ describe('Most status moves', function () {
 		assert.strictEqual(battle.p2.active[0].boosts['accuracy'], -1);
 	});
 
-	it('should not ignore type immunities', function () {
+	it('should fail when the opposing Pokemon is immune to the status effect it sets', function () {
```

Comment:
Despite how long "But it failed!" has been the message for this on PS, it's actually supposed to be the immunity message ("It doesn't affect POKEMON..."). "But it failed!" should only show up if one of these moves is used on a substitute. Nothing wrong with the way this test was set up before, but if it's changed to this and the message changes later it'll fail.